### PR TITLE
[W-11102773] Recurring Donation headers and Next Donation format

### PR DESCRIPTION
**Pull request by maurimart**
- Change custom labels for Recurring donation table component and modify Next Donation format.

# Critical Changes

# Changes

### CODE REVIEW

**Comment by maurimart** [comment]
[W-11102773](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000wR83eYAC/view)

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -5865,20 +5865,20 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>Amount</value>
     </labels>
     <labels>
-        <fullName>dateStarted</fullName>
+        <fullName>firstDonation</fullName>
         <categories>Gift Entry</categories>
```
Comment (lparrott) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:5869]: Let's use a more accurate Category, 'Donors' or 'Recurring-Donations', or come up with a new one.

File: force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js [hunk 2]
```diff
@@ -20,68 +20,63 @@ export default class RecurringDonationTable extends LightningElement {
     
     data = [{
```
Comment (lparrott) [inline force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js:21]: I know it's unrelated to the change, but is this test data? If so, why is it in the component itself?

**Review by gerardorodriguezsf — APPROVED** [review]
LGTM

**Review by gerardorodriguezsf — APPROVED** [review]

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (maurimart) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml]: good suggestion! changed!

File: force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js (hunk 2 above)
Comment (maurimart) [reply force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js:21]: Actually, they are only design data, they are not intended to be used in tests.

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 3]
```diff
@@ -5858,51 +5858,43 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
     </labels>
     <labels>
         <fullName>commonAmount</fullName>
-        <categories>Gift Entry</categories>
+        <categories>Recurring Donations</categories>
```
Comment (andrewyu-salesforce) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:5861]: `commonAmount` is an already existing custom label and is also being used in Gift Entry. You can leave the categories unchanged.

File: force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js (hunk 2 above)
Comment (andrewyu-salesforce) [reply force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js:21]: What do you mean by design data? Is it like an example use case that will be displayed to the end-user?

File: force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js [hunk 4]
```diff
@@ -20,68 +20,63 @@ export default class RecurringDonationTable extends LightningElement {
...
         "amount": 43,
         "paymentMethod": "laser",
         "frequency": "Monthly on the 22nd",
         "status": "Active",
-        "endDate": "12/10/2023",
-        "lastDonationDate": "4/10/2023",
-        "nextInstallment": "2 of 5 installments paid. Next on 2/22/2022",
-        "lastUpdated": "4/01/2023"
+        "mostRecenDonation": "12/10/2023",
+        "nextDonation": "Paid Donation: 3 of 6. Next Donation: 12/20/2022.",
```
Comment (andrewyu-salesforce) [inline force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js:42]: Next Donation should go to next line

File: force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js [hunk 5]
```diff
@@ -20,68 +20,63 @@ export default class RecurringDonationTable extends LightningElement {
...
         "amount": 91,
         "paymentMethod": "diners-club-enroute",
         "frequency": "Monthly on the 22nd",
         "status": "Active",
-        "endDate": "12/10/2023",
-        "lastDonationDate": "4/10/2023",
-        "nextInstallment": "10/14/2023",
-        "lastUpdated": "4/01/2023"
+        "mostRecenDonation": "12/10/2023",
+        "nextDonation": "Paid Donation: 2 of 10. Next Donation: 11/11/2022.",
```
Comment (andrewyu-salesforce) [inline force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js:66]: next donation should go to next line

File: force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js [hunk 6]
```diff
@@ -20,68 +20,63 @@ export default class RecurringDonationTable extends LightningElement {
...
         "amount": 10,
         "paymentMethod": "diners-club-us-ca",
         "frequency": "Monthly on the 22nd",
         "status": "Active",
-        "endDate": "12/10/2023",
-        "lastDonationDate": "4/10/2023",
-        "nextInstallment": "2 of 5 installments paid. Next on 2/22/2022",
-        "lastUpdated": "4/01/2023"
+        "mostRecenDonation": "12/10/2023",
+        "nextDonation": "Paid Donation: 6 of 8. Next Donation: 12/22/2022.",
```
Comment (andrewyu-salesforce) [inline force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js:78]: next donation should go to next line

**Review by andrewyu-salesforce — COMMENTED** [review]
@maurimart I left some initial feedback for the PR

File: force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js (hunk 4 above)
Comment (maurimart) [reply force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js]: done!

File: force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js (hunk 2 above)
Comment (maurimart) [reply force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js:21]: That's right @andrewyu-salesforce , sorry for the confusion, it's mock data, but it's not currently being used for testing, just to simulate the data and how the user should see it.
That is why it is not in a separate file or in the folder corresponding to the test for LWC.

**Pull request merged**